packages/manager/apps/hpc-vmware-managed-vcd/src/pages/listing/datacentres/Datacentres.constants.ts

```diff
@@ -1,2 +1,3 @@
 export const DEDICATED_PATH = 'dedicated';
 export const VRACK_PATH = 'vrack';
+export const VRACK_ONBOARDING_PATH = 'onboarding';
```

Comment:
Why don't use `packages/manager/apps/hpc-vmware-managed-vcd/src/routes/routes.constant.ts` to resolve onboarding path ?